docs/pages/getting-started/5-irma-contract.rst

```diff
@@ -0,0 +1,343 @@
+        translations: {
+          header: "Sign your contract"
+        },
+
+        // Back-end options
+        session: {
+          // Point to your web backend
+          url: '/web/auth',
+
+          // The request that will be send to the backend:
+          start: {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(this.some_data)
+          },
+
+          // required to translate Nuts specific return values
+          mapping: {
+            sessionPtr:      r => r.sessionPtr.clientPtr,
+            sessionToken:    r => r.sessionID
+          }
+        }
+      };
+
+      // we'll use the popup frontend
+      let irmaPopup = irma.newPopup(options);
+
+      // start the interaction
+      irmaPopup.start()
+          .then(result => {
+            console.log("success!")
+            console.log(response)
+          })
+          .catch(error => {
+            if (error === 'Aborted') {
+              console.log('Aborted');
+              return;
+            }
+            console.error("error", error);
+          })
+          .finally(() => irmaPopup = irma.newPopup(options));
+    }
+
+Lets break this down into parts.
+
+.. code-block:: javascript
+
+    // Developer options
+    debugging: true,
+
+Is used to enabling debugging. The IRMA library will output more information helpful for development.
+
+.. code-block:: javascript
+
+    // Front-end options
+    language: 'en',
+
+    // customize textual components
+    translations: {
+      header: "Sign your contract"
+    },
+
+Sets the language to english which will set some default textual representations on the IRMA web component.
+The ``translations`` configuration option can be used to change each of the textual representation on the IRMA web component.
+In this case, only the header is changed.
+
+.. code-block:: javascript
+
+    // Back-end options
+    session: {
+      // Point to your web backend
+      url: '/web/auth',
+
+      // The request that will be send to the backend:
+      start: {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(this.some_data)
+      },
+
+      // required to translate Nuts specific return values
+      mapping: {
+        sessionPtr:      r => r.sessionPtr.clientPtr,
+        sessionToken:    r => r.sessionID
+      }
+    }
+
+The ``session`` object contains all the technical parts to connect the IRMA JS library to your backend.
+The contents of the ``start`` object configures the initial request to start a signing session. You can control the type of request and the contents.
+In this case, some data from the frontend is sent as JSON. This is optional and no particular data is required.
+The ``url``, in this case ``/web/auth``, must be set in such a way that two backend services are available to the frontend.
```

Comment:
this is a complex way to say that everything under `<url>/session` must be accessible by the frontend?